src/components/cateSort.tsx

```diff
@@ -2,31 +2,26 @@ import { Link } from 'gatsby'
 import * as React from 'react'
 import Showletter from './showletter'
 
-
-
-interface CateSortProps{
-    readonly currentIndex : string
-    readonly preIndex : string
-    readonly numOfBlog : number
+interface CateSortProps {
+  readonly currentIndex: string
+  readonly preIndex: string
+  readonly numOfBlog: number
 }
 
-
-export class CateSort extends React.Component<CateSortProps,{}>{
-    render(){
-        
-        const { currentIndex } = this.props
-        const { preIndex } =this.props
-        const {numOfBlog} =this.props
-    return(
-        <div>
-        
-        { currentIndex.charAt(0)!==preIndex.charAt(0) && <Showletter letter={currentIndex}/>}
-        <Link to={currentIndex}>{currentIndex}
-            </Link> : {numOfBlog}.
-        </div>
+export class CateSort extends React.Component<CateSortProps, {}> {
+  render() {
+    const { currentIndex } = this.props
+    const { preIndex } = this.props
+    const { numOfBlog } = this.props
```

Comment:
No need to change in this PR, but  this can also be collapsed into a single variable declaration
```jsx
const { currentIndex, preIndex, numOfBlog } = this.props
```

Reply:
Ok, let me fix this.